src/fillTank.test.js

```diff
@@ -1,11 +1,167 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it(`should be declared`, () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it(`shouldn't return anything `, () => {
+    const customer = {
+      money: 2000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 3,
+      },
+    };
+
+    expect(fillTank(customer, 5, 5)).toBeUndefined();
+  });
 
+  it(`should fill all the tank if amount is undefined`, () => {
+    const customer = {
+      money: 2000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 3,
+      },
+    };
+
+    fillTank(customer, 10);
```

Comment:
The test case is intended to check the behavior when the amount is undefined, but the `fillTank` function is called with two arguments. Ensure that the function is called with the correct number of arguments to match the test description.